simpleflow/swf/process/decider/base.py

```diff
@@ -42,33 +58,34 @@ def __init__(self, executors, domain, task_list, nb_retries=3,
         behind this is to limit operational burden by having a single service
         handling multiple workflows.
 
-        :param executors: executors handling workflow executions.
-        :type  executors: list[simpleflow.swf.executor.Executor]
+        :param workflow_executors: executors handling workflow executions.
+        :type  workflow_executors: list[simpleflow.swf.executor.Executor]
 
         """
         self._workflow_name = '{}'.format(','.join(
             [
-                ex.workflow.name for ex in executors
+                ex.workflow.name for ex in workflow_executors
                 ]))
 
         # Maps a workflow's name to its definition.
         # Used to dispatch a decision task to the corresponding workflow.
-        self._workflows = {
-            executor.workflow.name: executor for executor in executors
+        self._workflow_executors = {
+            executor.workflow.name: executor for executor in workflow_executors
             }
 
         if not task_list:
-            task_list = executors[0].workflow.task_list
+            task_list = workflow_executors[0].workflow.task_list
 
         # All executors must have the same domain and task list.
-        for ex in executors[1:]:
+        for ex in workflow_executors:
             if ex.domain.name != domain.name:
                 raise ValueError(
                     'all workflows must be in the same domain "{}"'.format(
                         domain.name))
             if ex.workflow.task_list != task_list:
-                raise ValueError(
-                    'all workflows must have the same task list "{}"'.format(
+                # FIXME really needed?
```

Comment:
Yes. I had this problem at some time in the past, but now I think it was because I specified a task list on the command line or something... Will put the error back